dtde/config.py

```diff
@@ -26,3 +27,45 @@ def get() -> configparser.ConfigParser:
     if not global_config:
         load_config()
     return global_config
+
+
+class ConfigObject:
```

Comment:
I think we should have 1 object like that per config file. So maybe rename it to be ClusterConfig. Might also be worth it to make a parent class ConifgContainer that handle the reading of a file from yaml to an dict